var/spack/repos/builtin/packages/flcl/package.py

```diff
@@ -11,18 +11,23 @@ class Flcl(CMakePackage):
 
     homepage = "https://github.com/kokkos/kokkos-fortran-interop"
     git      = "https://github.com/kokkos/kokkos-fortran-interop.git"
-    url      = "https://github.com/kokkos/kokkos-fortran-interop/releases/download/0.3/0.3.tar.gz"
+    url      = "https://github.com/kokkos/kokkos-fortran-interop/releases/download/0.5.0/flcl-0.5.0.tar.gz"
 
     maintainers = ['womeld', 'agaspar']
 
     version('develop', branch='develop')
     version('0.3', sha256='0586b981b976588d8059e5bf1bf71fb5a7153ea950c7e2b562a3d812fefee56e')
```

Comment:
The URL change changes the checksum for this version to the following:

```suggestion
    version('0.3', sha256='fc18c8fa3ae33db61203b647ad9025d894612b0faaf7fe07426aaa8bbfa9e703')
```

Reply:
Confirmed the URLs for the new versions are correct.